src/Appwrite/Utopia/Response/Filters/Select.php

```
@@ -0,0 +1,36 @@
+    {
+    }
+
+    /**
+     * Filter response content to only include selected attributes.
+     *
+     * @param array $content
+     * @param string $model
+     *
+     * @return array
+     */
+    public function parse(array $content, string $model): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$model'. (undefined)

(UnusedFormalParameter)